addons/website/static/src/scss/website.scss

```diff
@@ -1511,8 +1511,11 @@ ul.o_checklist > li.o_checked::after {
 .o_animate_preview {
     visibility: visible;
 }
-.o_wanim_overflow_x_hidden {
+.o_wanim_overflow_xy_hidden {
     overflow-x: hidden !important;
+    .o_footer_slideout {
+        overflow-y: hidden !important;
+    }
```

Comment:
Shouldn't it be on the `wrapwrap` instead? some animations can start from below on the last block of the page and also create height flickering... (probably even more visible on a page without a footer)

Reply:
> Shouldn't it be on the wrapwrap instead?

No, saying that the Y overflow on the `wrapwrap` is hidden means that you can't scroll the page (during element animations) which we don't want.

> some animations can start from below on the last block of the page and also create height flickering

You're right, if you're not in a slideout footer (or there is no footer at all), the page height will flicker which is not ideal... the purpose of this is to make the animations work properly with the new way of calculating if an element is visible when that element is in a slideout footer. The new way of calculating depends on the [scrollHeight](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollHeight) of the page which changes in case an element appears from the bottom of the page. When the element is not in a footer slideout, we check only if it is in the viewport which does not depend on the scrollHeight. I could still fix the flicker of the page height but I'm not sure it's worth it (since animations are not impacted), I would have to detect that the element is low enough in the page, disable the Y overflow on one of its ancestors big enough for the animation to be fully visible,... Maybe I'm wrong about something, if so don't hesitate :)

Also note that I have the suspicion that the option to hide the footer is no longer working on master. I will double check and create a task if needed

Reply:
Indeed I spoke too fast for the overflow on the `wrapwrap`. 
And yes, I don't think it's worth it as you say, so I would leave it like that..